not-to-release/sources/answers/20111107144339AA0qw5S_ans.xml.conllu

```diff
@@ -387,7 +387,7 @@
 10	really	really	ADV	RB	_	11	advmod	11:advmod	_
 11	attended	attend	VERB	VBN	Tense=Past|VerbForm=Part|Voice=Pass	0	root	0:root	_
 12	by	by	ADP	IN	_	13	case	13:case	_
-13	protestants	protestants	PROPN	NNPS	Number=Plur	11	obl	11:obl:by	_
+13	protestants	protestant	NOUN	NNS	Number=Plur	11	obl	11:obl:by	_
```

Comment:
Protestant/Catholic should be PROPN right?

Reply:
The UD guidelines say "A proper noun is a noun (or nominal content word) that is the name (or part of the name) of a specific individual, place, or object". So is a plural like protestants still an proper noun? I thought not, that is why I propose to retag it as NOUN

Reply:
I would not make a distinction based on singular vs. plural. "Protestants" could refer to all individuals, or a subset of individuals, who identify as Protestant. Likewise, a plural person name ("John Smiths") should still be PROPN.

Reply:
But everything could refer to an individual, e.g. "the president said..." in a given context can refer to a president of a given country. Is this tagged PROPN as well?

Reply:
I suppose there's some gray area with offices like "president". I would probably tag it as NOUN unless it's title within a specific person's name ("President Barack Obama"). UD strives to make tagging decisions on a fairly lexical basis and not require a ton of contextual interpretation about referents.